manifest.yml.j2

```diff
@@ -64,6 +64,7 @@
   'notify-delivery-worker-letters': {'memory': '2G'},
   'notify-delivery-worker-retry-tasks': {},
   'notify-delivery-worker-internal': {},
+  'notify-delivery-worker-broadcasts': {},
```

Comment:
can put `'instances': {'preview': 2, 'staging': 2, 'production': 2}` in here if we want to define instances without adding to autoscaler.

but do we need to configure autoscaler anyway to get metrics though?

Reply:
Will configure it in the autoscaler to give us metrics so won't specify the instances in here. Given that, I think this is ready for a final review please Leo